query_optimizer/ExecutionGenerator.hpp

```diff
@@ -196,6 +196,13 @@ class ExecutionGenerator {
       const CatalogRelation **catalog_relation_output,
       serialization::InsertDestination *insert_destination_proto);
 
+  void createPartitionedTemporaryCatalogRelation(
+      const physical::PhysicalPtr &physical,
+      const CatalogRelation **catalog_relation_output,
+      serialization::InsertDestination *insert_destination_proto,
+      const attribute_id part_attr_id,
+      const std::size_t num_partitions);
```

Comment:
According to [the code style](https://google.github.io/styleguide/cppguide.html#Function_Parameter_Ordering), we should put all input arguments before the output ones. In this case, the last two are the input ones, so I would suggest the following refactor:

```
  void createPartitionedTemporaryCatalogRelation(
      const physical::PhysicalPtr &physical,
      const attribute_id part_attr_id,
      const std::size_t num_partitions,
      const CatalogRelation **catalog_relation_output,
      serialization::InsertDestination *insert_destination_proto);
```